kustomize/preprod/simple-app/deployment.yaml

```diff
@@ -22,6 +52,9 @@ spec:
               cpu: "250m"
           ports:
             - containerPort: 5000
+          # volumeMounts:
```

Comment:
We need to mount `env-volume` volume to the container that contain the .env. See `subPath` option

Reply:
Updated! Please re-review and send me feedbacks. Thanks!